test/emqx_connection_SUITE.erl

```diff
@@ -386,6 +394,41 @@ t_get_conn_info(_) ->
                                     }, SockInfo)
               end).
 
+t_oom_shutdown(init, Config) ->
+    ok = snabbkaffe:start_trace(),
+    snabbkaffe:fix_ct_logging(),
```

Comment:
This call can be removed. I added this function because I don't like how CT treats the logs, but in the projects that already rely on the common test logging it can be confusing.

Reply:
fixed